tests/test_amda_parameter.py

```diff
@@ -0,0 +1,86 @@
+#!/usr/bin/env python
+# -*- coding: utf-8 -*-
+
+"""Tests for `amda` package parameter getting functions."""
+
+import unittest
+import os
+from datetime import datetime, timezone
+from speasy.amda import AMDA, load_csv
+from speasy.common.variable import SpeasyVariable
+from ddt import ddt, data
+import numpy as np
+
+from speasy.amda.dataset import Dataset
+from speasy.amda.parameter import Parameter
+
+@ddt
+class SimpleRequest(unittest.TestCase):
+    def setUp(self):
+        self.ws = AMDA()
+        self.start=datetime(2000,1,1,1,1)
+        self.stop=datetime(2000,1,1,1,2)
+        self.data = self.ws.get_parameter("imf", self.start, self.stop)
+        self.dataset = self.ws.get_dataset("ace-imf-all", self.start, self.stop)
+
+
+    def tearDown(self):
+        pass
+    # check that the data is not None
```

Comment:
You don't need to comment if it doesn't add information that the code doesn't provide.